packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -249,14 +249,14 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
     let axisType: XAxisTypes | null = null;
     let pointToHighlight: string | Date | number | null = null;
     let index: null | number = null;
+    axisType = getTypeOfAxis(lineChartData![0].data[0].x, true) as XAxisTypes;
```

Comment:
>axisType [](http://example.com/codeflow?start=4&length=8)

what is the logic change here. axisType is not even getting used in the first 2 if else if blocks. 
lineChartData![0].data[0] could also be undefined in this line.

Reply:
a single data point causes the bisect() call return one value among d0 and d1 as undefined
So the correct axis type is never set for such cases. We need to explore more cases where this can potentially turn undefined and does it break the component in any case.